### ISSUE TITLE
NoClassDefFoundError: org/swellrt/model/WaveConversationUtils

### ISSUE THREAD

**Issue by atd** [P2Pvalue/swellrt#9]
Starting the last version of SwellRT jar throws the following error:

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/swellrt/model/WaveConversationUtils
        at java.lang.Class.getDeclaredMethods0(Native Method)
        at java.lang.Class.privateGetDeclaredMethods(Class.java:2615)
        at java.lang.Class.getDeclaredMethods(Class.java:1860)
        at com.google.inject.spi.InjectionPoint.getInjectionPoints(InjectionPoint.java:662)
        at com.google.inject.spi.InjectionPoint.forInstanceMethodsAndFields(InjectionPoint.java:356)
        at com.google.inject.internal.ConstructorBindingImpl.getInternalDependencies(ConstructorBindingImpl.java:151)
        at com.google.inject.internal.InjectorImpl.getInternalDependencies(InjectorImpl.java:585)
        at com.google.inject.internal.InjectorImpl.cleanup(InjectorImpl.java:543)
        at com.google.inject.internal.InjectorImpl.initializeJitBinding(InjectorImpl.java:529)
        at com.google.inject.internal.InjectorImpl.createJustInTimeBinding(InjectorImpl.java:847)
        at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:772)
        at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:761)
        at com.google.inject.internal.InjectorImpl.getJustInTimeBinding(InjectorImpl.java:256)
        at com.google.inject.internal.InjectorImpl.getBindingOrThrow(InjectorImpl.java:205)
        at com.google.inject.internal.InjectorImpl.getInternalFactory(InjectorImpl.java:853)
        at com.google.inject.internal.InjectorImpl.getProviderOrThrow(InjectorImpl.java:967)
        at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:1000)
        at com.google.inject.internal.InjectorImpl.getProvider(InjectorImpl.java:961)
        at com.google.inject.internal.InjectorImpl.getInstance(InjectorImpl.java:1013)
        at org.waveprotocol.box.server.ServerMain.run(ServerMain.java:177)
        at org.waveprotocol.box.server.ServerMain.main(ServerMain.java:147)
Caused by: java.lang.ClassNotFoundException: org.swellrt.model.WaveConversationUtils
        at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
        at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
        at java.security.AccessController.doPrivileged(Native Method)
        at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
        ... 21 more
```

**Comment by atd** [comment]
@atfornes this is probably the cause of Pear2Pear not starting. I get the same error both in the docker container and in wave.p2pvalue.eu

**Comment by atfornes** [comment]
It seams that  2169b73 did not fixed the error :(

**Comment by atfornes** [comment]
@pablojan, 

a54a5e5 introduced the dependencie WaveConversationUtils class in classes PersistenceModule, MongoDbIndexStore and MongoDbProvider.

**Comment by atfornes** [comment]
As @pablojan explained, after cleaning with ```ant -f build-swellrt.xml clean``` the problem is solved thanks to commit 2169b73.

Thanks!

**Comment by swartz-p2pvalue** [comment]
What worked for me was `ant clean dist`
